### DISCUSSION TITLE
Any timeline on a JSON type?

### DISCUSSION THREAD

**Discussion by venables** [prisma/prisma#2166] [Q&A] [answered]
I saw via https://github.com/prisma/prisma/issues/186 there's plans to support JSON, but is there a sense in the roadmap as to where/when this will happen.  Is the current workaround using a String and parsing/stringifying manually?

**Comment by janpio** [comment, marked as answer]
> but is there a sense in the roadmap as to where/when this will happen.

No ETA, sorry. But we expect to work on this soon.

> Is the current workaround using a String and parsing/stringifying manually?

That should work, yes.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
